src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -589,12 +602,15 @@ ProcessCLRException(IN     PEXCEPTION_RECORD   pExceptionRecord,
 
     Thread* pThread         = GetThread();
 
+    // On x86 we don't have dispatcher context
```

Comment:
I believe this can go away, with the recent removal of the ProcessCLRException personality routine from asm helpers, we should never reach this place for non-managed code frames. 
But let me double check that we don't get here for some dynamically generated helpers that are not managed code. I don't think we would, but I want to be sure. So let's keep this as is and I'll remove it in a separate PR.